src/components/project/project.service.ts

```diff
@@ -377,6 +378,37 @@ export class ProjectService {
       },
     });
 
+    if (filter.mine) {
```

Comment:
Filters need to be applied at the database level so pagination is accurate.

Reply:
yes done